boot/build_test.go

```diff
@@ -637,4 +629,45 @@ Spring-Boot-Lib: BOOT-INF/lib
 		})
 
 	})
+
+	context("when there is a non-exploded jar passed to the buildpack", func() {
+
+		it.Before(func() {
+			t.Setenv("BP_SPRING_CLOUD_BINDINGS_DISABLED", "true")
+			os.Remove(filepath.Join(ctx.Application.Path, "META-INF"))
+		})
+
+		it("finds and extracts a jar that exists", func(){
+
+	    	Copy("cds","spring-app-3.3-no-dependencies.jar", "")
```

Comment:
```suggestion
			Copy("cds","spring-app-3.3-no-dependencies.jar", "")
```

should be indented like the line below (Expect); with 3 tabs